kwc-auth.html

```diff
@@ -357,9 +358,23 @@
                  */
                 firstName: {
                     type: String,
+                    value: null,
                     observer: '_onChangeFirstName'
                 },
                 /**
+                 * Default user first name. This is
+                 * used if no first name is given
+                 * @type {String}
+                 */
+                defaultFirstName: {
+                    type: String,
+                    value: "your kid"
```

Comment:
This is for sure not the right copy. We actively don't use the word kid :)

Reply:
This is directly from the code base: https://github.com/KanoComputing/web-components/blob/master/kano-auth/kano-auth.html#L513

Reply:
Considering this should never happen since the first name is mandatory, I guess it's fine... Could you have a chat with Martin about it?

Reply:
The first name is not mandatory. It is never used either!

Reply:
First name is not mandatory and in fact never used except for in the text of the following auth modal view.